riscv-asm.md

```diff
@@ -184,6 +184,150 @@ Directive    | Arguments                      | Description
 .zero        | integer                        | zero bytes
 .variant_cc  | symbol_name                    | annotate the symbol with variant calling convention
 
+### <a name=.option></a> `.option`
+
+#### `rvc`/`norvc`
+
+This option will be deprecated soon once `.option arch` is widely implemented on
+main stream open source toolchains.
+
+Enable/disable the C-extension for the following code region, this option is equivalent
+to `.option arch, +c`/`.option arch, -c`, but widely support by older version of
+toolchain.
+
+NOTE: `.option rvc` might set the `EF_RISCV_RVC` in the ELF flags on specific
+toolchains, that might cause linker convert instruction into compressed
+instruction on unexpected region.
+
+NOTE: The difference between `.option rvc`/`.option norvc` and
+`.option arch +c`/`.option arch -c` won't set `EF_RISCV_RVC` in the ELF flags.
```

Comment:
NOTE: There is a difference between `.option rvc`/`.option norvc` and
`.option arch +c`/`.option arch -c`. The latter won't set `EF_RISCV_RVC` in the ELF flags.